examples/postgres/docker-compose.yaml

```diff
@@ -0,0 +1,40 @@
+version: "3.7"
+services:
+
+  proxy:
+    image: envoyproxy/envoy-dev:latest
+    volumes:
+      - ./envoy.yaml:/etc/envoy.yaml
+    networks:
+      envoymesh:
+        aliases:
+          - envoy
+    expose:
+      - "1999"
+      - "8001"
+    ports:
+      - "1999:1999"
+      - "8001:8001"
+    command: "/usr/local/bin/envoy -c /etc/envoy.yaml -l debug"
+
+  postgres:
+    image: postgres:latest
+    networks:
+      envoymesh:
+        aliases:
+          - postgres
+    environment:
+      # WARNING! Do not use it on production environments because this will
+      #          allow anyone with access to the Postgres port to access your
+      #          database without a password, even if POSTGRES_PASSWORD is set.
+      #          See PostgreSQL documentation about "trust":
+      #          https://www.postgresql.org/docs/current/auth-trust.html
+      POSTGRES_HOST_AUTH_METHOD: trust
+    expose:
```

Comment:
i dont think either `expose` or `ports` is necessary as the containers communicate internally

Reply:
Totally agree. Fixed!